lib/db.c

```diff
@@ -3748,6 +3748,34 @@ grn_table_add_by_key(grn_ctx *ctx, grn_obj *table, grn_obj *key, int *added)
   return id;
 }
 
+grn_id
+grn_table_get_without_normalize(grn_ctx *ctx,
```

Comment:
I don't want to add this. If we add this, we want to add `grn_table_cursor_open_without_nromalize()`.

I think that this is a `ii` problem. `ii` should not re-get token from token id. `ii` should use source of a token instead of re-getting token. We may be able to use `grn_token_get_source_offset()` and `grn_token_get_source_length()` for it. (We can add `grn_token_get_source()` for convenient.)